NextJS API Integration Project Files (#55)
dequelabs/axe-devtools-html-api-examples: Node/nextjs/README.md

# axe DevTools NextJS API Example Project

## Prerequisites

- npm
- NodeJS (6.10 or higher)

## Installation information

In order to use this example project, you will need to clone the repository from GitHub. Once you've done that, in your terminal, navigate from the project root back to this example.

```

> cd node/nextjs

Reviewer: ```suggestion
cd Node/nextjs
```

```

Next, you'll need to install the dependencies for this project including axe DevTools Browser and the DevTools Reporter. They are already listed in `package.json`, but access to Deque's private registry is required to install them. To do so, follow [this guide](https://axe-devtools-html-docs.deque.com/reference/node/browser/install-agora.html) on the Deque documentation site. Once your access to Deque's private registry has been configured, dependencies for this project can be installed as normal through yarn or npm.

```
npm install
```

## Running the test cases

Use this command to run the example

```
npm test
```

## Project Setup

> The sample test file is held within the `__test__` directory. This file contains an example of how to set up the axe DevTools Browser and Reporter APIs, test for accessibility, and generate formatted results.

Reviewer: ```suggestion
The sample test file is held within the `__tests__` directory. This file contains an example of how to set up the axe DevTools Browser and Reporter APIs, test for accessibility, and generate formatted results.
```

> Once the sample project has been run, formatted results can be found in the `a11y-results` folder. The folder will contain the raw JSON results as well as the html, csv, and xml report formats. There will be one additional `html` file which is an executive summary report aggregating results from all scans into one page.

Reviewer: ```suggestion
Once test suite has completed formatted results can be found in the `a11y-results` folder. This will contain the raw JSON axe results as well as the html, csv, and xml report formats. There will be one additional `html` file which is an executive summary report aggregating results from all scans into one page.
```
